x-pack/plugins/ml/public/application/components/custom_urls/custom_url_editor/utils.ts

```diff
@@ -243,7 +315,7 @@ function buildDiscoverUrlFromSettings(settings) {
 
   const urlValue = `discover#/?_g=${_g}&_a=${_a}`;
 
-  const urlToAdd = {
+  const urlToAdd: any = {
```

Comment:
you can use `KibanaUrlConfig` type here 

```suggestion
  const urlToAdd: KibanaUrlConfig = {
```

Reply:
Updated type in 642ce6dbe70bb719b6416e5a61184a6b966b71d7